packages/components/src/components/wizard/wizard.scss

```diff
@@ -0,0 +1,42 @@
+.wizard {
+    width: inherit;
+    height: inherit;
+
+    &__step {
+        width: 100%;
+        align-items: center;
+    }
+    &__header {
+        display: flex;
+        flex-direction: row;
+        justify-content: space-between;
+    }
+    &__placeholder {
+        width: 6.3rem;
+    }
+    &__btn {
+        display: flex;
+        font-size: 1.4rem;
```

Comment:
for text we usually use the mixin `typeface` to combine the text properties into one line

otherwise LGTM

Reply:
Done